src/main.rs

```diff
@@ -4,12 +4,12 @@ mod args;
 use aoc::*;
 use args::*;
 use clap::Parser;
-use home::home_dir;
 use std::fs::read_to_string;
 use std::path::PathBuf;
 use std::process::exit;
 
-const SESSION_COOKIE_FILE: &str = ".adventofcode.session";
+const SESSION_COOKIE_FILE: &str = "adventofcode.session";
```

Comment:
I think it's a bit confusing to have different names for different directories. I'd suggest keeping the dot for both.

Reply:
It's highly unusual to have a hidden config file inside of config in my experience.

E.g. it is `~/.gitignore` and `~/.config/git/ignore`.

Reply:
Don't think git-ignore is a good example as it is not a sensitive file (it is only hidden inside a live Git repo to reduce clutter).

Reply:
> Don't think git-ignore is a good example as it is not a sensitive file (it is only hidden inside a live Git repo to reduce clutter).

This has nothing to do with sensitivity, and for that concern `.config` is already hidden.

Hiding files in home is more for having less clutter than actually hiding them from misuse.

And for a different example, it is `~/.vim` but `~/.config/nvim`

Reply:
Ok fair enough